src/app/forms/templates/general/defect.template.ts

```diff
@@ -91,51 +92,52 @@ export const DefectsTpl: FormNode = {
                               name: 'vertical',
                               label: 'Vertical',
                               type: FormNodeTypes.CONTROL,
-                              disabled: true
+                              disabled: false
```

Comment:
I thought we defaulted to false for disabled, meaning none of these would be needed?